Caboodle/DataTransfer/DataTransfer.uwp.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.Foundation;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class DataTransfer
+    {
+        static ShareTextRequest shareRequest;
+
+        public static Task ShowShareUI(ShareTextRequest request)
```

Comment:
I think the method may be better named `RequestRansfer` rather than show share ui. Just my preference as I don't like the `UI` casing as well as we are talking transfer in the class name, and then switch to share in the method.

Reply:
I think just "Request" would be good..

DataTransfer.Request("text")

Reply:
and should be REquestAsync